grafana/dashboard.json

```diff
@@ -486,14 +486,14 @@
                 "allValue": null,
                 "current": {},
                 "datasource": "${DS_PROMETHEUS}",
-                "definition": "label_values(instance)",
+                "definition": "label_values(nginx_up, instance)",
```

Comment:
Why do we have `nginx_up` as a label_value here? I wonder if this is really needed as it doesn't seem `nginx_up` is a label itself. Can you elaborate the use of this? Thanks!

Reply:
Sure. `nginx_up` is required for instance variable to filter just those instances that have nginx_exporter metrics, otherwise it should show all instances that grafana sees.

Reply:
Yes you are absolutely right, I didn't think of the use case of having other services using `instance` label. 👍